general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -336,9 +336,10 @@ def submit_qubo(
         Args:
             qubo: A `qv.QUBO` object.
             target: The target to submit the qubo.
-            repetitions: Number of repetitions to use. Defaults to 1000.
-            method: An optional method parameter. Defaults to None.
-            maxout: An optional maxout paramter. Defaults to 1000.
+            repetitions: Number of times that the execution is repeated before stopping.
+            method: The parameter specifying method of QUBO solving execution. Defaults to none.
+            maxout: A parameter that specifies the max number of output solutions.
```

Comment:
we should add `gss.validation.validate_integer_param(maxout)` alongside the other validators below

Reply:
done.